src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -1114,4 +1128,37 @@ public SequenceFilePair getSequenceFilePair() {
 			return sequenceFilePair;
 		}
 	}
+
+	/**
+	 * UI Model to return a single end sequence file with its sample
```

Comment:
I was just duplicating what was done above in the paired version of the class.  I refactored this down to a single private class now in 2da4fa3.  I didn't put it into a separate `dto` class because the constructor is using some service calls to pick up the associated sample.

Note this will all get removed anyway when the new analysis pages are done.